packages/moonstone/VideoPlayer/FeedbackTooltip.js

```diff
@@ -138,9 +171,12 @@ const FeedbackTooltipBase = kind({
 	},
 
 	render: ({children, noFeedback, playbackState, playbackRate, thumbnailComponent, ...rest}) => {
-		delete rest.visible;
+		delete rest.duration;
+		delete rest.formatter;
 		delete rest.thumbnailDeactivated;
 		delete rest.thumbnailSrc;
+		delete rest.visible;
```

Comment:
We should delete `orientation` and `proportion` as well as other props that are passed from the render function of `ui/Slider`